docs-source/docs/modules/how-to/examples/shopping-cart-service-cassandra-java/pom.xml

```diff
@@ -15,99 +15,106 @@
         </license>
     </licenses>
 
-    <!-- tag::akka-persistence-cassandra[] -->
+    <!-- tag::pekko-persistence-cassandra[] -->
     <properties>
-        <!-- end::akka-persistence-cassandra[] -->
+        <!-- end::pekko-persistence-cassandra[] -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <akka-grpc-maven-plugin.version>2.1.3</akka-grpc-maven-plugin.version>
+        <pekko-grpc-maven-plugin.version>1.0.1</pekko-grpc-maven-plugin.version>
+        <pekko.version>1.0.2</pekko.version>
+        <pekko.connectors.version>1.0.1</pekko.connectors.version>
+        <pekko.projection.version>1.0.0</pekko.projection.version>
+        <pekko.grpc.version>1.0.0</pekko.grpc.version>
+        <pekko.persistence.jdbc.version>1.0.0</pekko.persistence.jdbc.version>
+        <pekko.persistence.cassandra.version>1.0.0</pekko.persistence.cassandra.version>
+        <pekko.connectors.kafka.version>1.0.0</pekko.connectors.kafka.version>
+        <pekko.management.version>1.0.0</pekko.management.version>
         <scala.binary.version>2.13</scala.binary.version>
         <!-- needs to be defined to allow for overriding through mvn exec:exec -DAPP_CONFIG=local1.conf -->
         <APP_CONFIG>application.conf</APP_CONFIG>
         <!-- Version of the Docker image is derived from git commit.
              It can be defined with -Dversion.number=0.1-SNAPSHOT if git is not used -->
         <version.number>${git.commit.time}-${git.commit.id.abbrev}</version.number>
-        <!-- tag::akka-persistence-cassandra[] -->
+        <!-- tag::pekko-persistence-cassandra[] -->
     </properties>
-    <!-- end::akka-persistence-cassandra[] -->
+    <!-- end::pekko-persistence-cassandra[] -->
 
-    <dependencyManagement>
```

Comment:
could you use the pekko-bom like I used in microservices tutorials for consistency? If you use that, then you don't need to set the pekko version on the jars that were `<groupId>com.typesafe.akka</groupId>`

Reply:
I didn't find it similar at first.  Get it now, thanks.
          <dependency>
                <groupId>org.apache.pekko</groupId>
                <artifactId>pekko-bom_${scala.binary.version}</artifactId>
                <version>${pekko.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>